src/goupile/server/admin.cc

```diff
@@ -211,7 +175,7 @@ T(R"(Usage: %!..+%1 init [option...] [directory]%!0)"), FelixTarget);
         const char *filename = Fmt(&temp_alloc, "%1%/goupile.ini", root_directory).ptr;
         files.Append(filename);
 
-        if (!WriteFile(DefaultConfig, filename))
+        if (!WriteFile(gDefaultConfigData, filename))
```

Comment:
`<prefix>Var<data>` will have the contents for `Var`, default prefix is `g` and `Data` is the suffix.